geowave-test/pom.xml

```diff
@@ -91,7 +91,7 @@
 									<type>war</type>
 									<overWrite>true</overWrite>
 									<outputDirectory>${project.build.directory}/geoserver</outputDirectory>
-									<excludes>**/commons-dbcp-1.4.jar,**/*1.7-SNAPSHOT.jar,**/*2.7-SNAPSHOT.jar,**/*20140915*.jar,**/*13-SNAPSHOT.jar</excludes>
+									<excludes>**/*1.7-beta.jar,**/*13-beta.jar,**/*2.7-beta.jar,**/commons-dbcp-1.4.jar,**/*1.7-SNAPSHOT.jar,**/*2.7-SNAPSHOT.jar,**/*20140915*.jar,**/*13-SNAPSHOT.jar</excludes>
```

Comment:
The same goes for the snapshot jars. Those are excluded to fix 2.5.2. We
need to get in contact with boundless to have them fix their jars.
On Jan 22, 2015 10:44 PM, "Chris Bennight" <notifications@github.com> wrote:

> In geowave-test/pom.xml
> <https://github.com/ngageoint/geowave/pull/191#discussion_r23430427>:
>
> > @@ -91,7 +91,7 @@
> >  									<type>war</type>
> >  									<overWrite>true</overWrite>
> >  									<outputDirectory>${project.build.directory}/geoserver</outputDirectory>
> > -									<excludes>**/commons-dbcp-1.4.jar,**/*1.7-SNAPSHOT.jar,**/*2.7-SNAPSHOT.jar,**/*20140915*.jar,**/*13-SNAPSHOT.jar</excludes>
> > +									<excludes>**/*1.7-beta.jar,**/*13-beta.jar,**/*2.7-beta.jar,**/commons-dbcp-1.4.jar,**/*1.7-SNAPSHOT.jar,**/*2.7-SNAPSHOT.jar,**/*20140915*.jar,**/*13-SNAPSHOT.jar</excludes>
>
> Probably remove this exclusion when the artifacts get fixed - shouldn't
> need it
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ngageoint/geowave/pull/191/files#r23430427>.
>